### ISSUE TITLE
Add to gallery of Jupyter Books

### ISSUE THREAD

**Issue by rzmk** [dathere/100.dathere.com#2]
There is a gallery of Jupyter Books available on the executable books site. We may add this book 100 exercises with qsv to the gallery by following the instructions here: https://executablebooks.org/en/latest/gallery/

This could also be beneficial for others even though it's incomplete. For example they may learn how to set up a similar notebook since the source code is available.

**Comment by a5dur** [comment]
Yeah, sounds like a very good idea. How about we add a qsv playground on the last page of the Jupyter notebook?

**Comment by rzmk** [comment]
@a5dur I added a Jupyter Lab playground notebook and a launch button on [qsv.dathere.com](https://qsv.dathere.com)'s home and nav bar.
